tests/ssl-opt.sh

```diff
@@ -1446,46 +1459,50 @@ run_test    "Max fragment length: enabled, default" \
 requires_config_enabled MBEDTLS_SSL_MAX_FRAGMENT_LENGTH
 run_test    "Max fragment length: enabled, default, larger message" \
             "$P_SRV debug_level=3" \
-            "$P_CLI debug_level=3 request_size=16385" \
+            "$P_CLI debug_level=3 request_size=$(expr $MAX_CONTENT_LEN + 1)" \
             0 \
-            -c "Maximum fragment length is 16384" \
-            -s "Maximum fragment length is 16384" \
+            -c "Maximum fragment length is $MAX_CONTENT_LEN" \
+            -s "Maximum fragment length is $MAX_CONTENT_LEN" \
             -C "client hello, adding max_fragment_length extension" \
             -S "found max fragment length extension" \
             -S "server hello, max_fragment_length extension" \
             -C "found max_fragment_length extension" \
-            -c "16385 bytes written in 2 fragments" \
-            -s "16384 bytes read" \
+            -c "$(expr $MAX_CONTENT_LEN + 1) bytes written in 2 fragments" \
```

Comment:
Please use shell arithmetic `$((MAX_CONTENT_LEN + 1))`, not `expr`. We don't support pre-POSIX Bourne shells, but we do try to support incomplete Unix environments which may have only `sh` and `grep`. Plus shell arithmetic is faster, especially in environments like Cygwin where fork is very slow.